packages/create-app/README.md

```diff
@@ -15,6 +15,7 @@ $ npx @backstage/create-app
 With a local clone of this repo, from the main `create-app/` folder, run:
 
 ```sh
+$ yarn
```

Comment:
In docs I think it's best to type out the entire `yarn install` for clarity

Reply:
> Explicit is better than implicit.

Good point, added in a54fdab36e32aa796963d901706a13e477fb6eb9 👍